data-source/json/test_default.json

```diff
@@ -28,19 +28,36 @@
                     "blocks": [
                         {
                             "type": "Question",
-                            "id": "number-question",
+                            "id": "number-question-one",
                             "question": {
                                 "answers": [
                                     {
-                                        "id": "answer",
+                                        "id": "answer-one",
                                         "mandatory": false,
                                         "type": "Number",
                                         "label": "Leave blank",
                                         "default": 0
                                     }
                                 ],
                                 "id": "question",
-                                "title": "Don't enter an answer. Zero will be shown on summary",
+                                "title": "Don't enter an answer. A default value will be used",
```

Comment:
Can we add a schema to test routing with a default answer?

Reply:
Added a new test.